partcad/tests/unit/test_adhoc_convert.py

```
@@ -0,0 +1,120 @@
+    input_file, input_format = temp_cad_file
+    output_file, output_format = temp_output_file
+
+    # Skip same-format conversion
+    if input_format == output_format:
+        pytest.skip(f"Skipping same-format conversion: {input_format} -> {output_format}")
+
+    convert_cad_file(input_file, input_format, output_file, output_format)
+    assert output_file.exists(), f"Conversion failed: {input_format} -> {output_format}"
+
+
+def test_convert_invalid_file():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_convert_invalid_file`

Add return type annotation: `None`

(ANN201)